application/config/version.php

```diff
@@ -12,7 +12,7 @@
  */
 
 $config['versionnumber'] = '6.1.2';
-$config['dbversionnumber'] = 606;
+$config['dbversionnumber'] = 607;
```

Comment:
Dev branch is already on version 608.

Reply:
I wouldnt artificially change it right now but instead do it during a merge